{
  "error": "cannot follow a user that is not on this instance"
}